swift/apple/FirezoneNetworkExtension/NetworkSettings.swift

```diff
@@ -30,6 +30,10 @@ class NetworkSettings {
   let tunnelAddressIPv4: String
   let tunnelAddressIPv6: String
   let dnsAddress: String
+    
+    // In theory we could update the MTU dynamically based on the network environment,
+    // but 1280 is guaranteed to work everywhere.
+  let tunnelMTU = NSNumber(1280)
```

Comment:
cc @roop -- On the Rust side, we try to read the mtu of the interface because we need it for the NAT traversal logic.